src/client/src/orangehrmClaimPlugin/components/ClaimExpenses.vue

```diff
@@ -226,9 +226,7 @@ export default {
       return computedHeaders;
     },
     formatedAmount() {
-      let value = this.totalAmount.replace(/,/g, '');
-      value = value.replace(/\B(?=(\d{3})+(?!\d))/g, ',');
-      return value;
+      return this.totalAmount.replace(/\B(?=(\d{3})+(?!\d))/g, ',');
```

Comment:
check this,
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString